clients/java/src/main/java/io/camunda/zeebe/client/api/search/response/ProcessDefinition.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.zeebe.client.api.search.response;
+
+public interface ProcessDefinition {
```

Comment:
This looks very similar to `io.camunda.zeebe.client.api.response.Process` that is used in the command clients. Should we use 2 different datamodels?

Reply:
Good catch! Will be handled in https://github.com/camunda/camunda/issues/23120